test/helper.rb

```diff
@@ -135,18 +192,35 @@ def self.run(reporter, options = {}) # :nodoc:
     super
   end
 
+  def after_teardown
+    return if passed? || skipped?
+    out_io unless Puma.jruby? or RUBY_VERSION < '2.3'
```

Comment:
What's different about jruby and Ruby 2.2?

Reply:
I'll double check.  Something must have happened, don't recall what.